### PR TITLE
AUdachina lab3

### PR DESCRIPTION
**Pull request by anastasiaudachina**

### CODE REVIEW

File: llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h [hunk 1]
```diff
@@ -16,4 +16,4 @@ namespace llvm {
 	};
 
 }
-#endif
+
```
Comment (geexie) [inline llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h:19]: @anastasiaudachina, why you've moved this to cpp file?

Comment (anastasiaudachina) [reply llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h:19]: @geexie it is from lab2. You asked me to move this part to the cpp file:
#ifndef LLVM_TRANSFORMS_UDACHINAPASS_H
#define LLVM_TRANSFORMS_UDACHINAPASS_H
#define DEBUG_TYPE "udachinaPass"

So I added #endif to follow the construction

Comment (geexie) [reply llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h:19]: I meant #define just `DEBUG_TYPE "udachinaPass"`  and keep 

```
#ifndef LLVM_TRANSFORMS_UDACHINAPASS_H
#define LLVM_TRANSFORMS_UDACHINAPASS_H
```
 in header as a [header guard](https://www.learncpp.com/cpp-tutorial/header-guards/)

**Review by geexie — APPROVED** [review]

File: llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h (hunk 1 above)
Comment (anastasiaudachina) [reply llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h]: @geexie I'm sorry, I misunderstood at first. I fixed it

**Pull request merged**